src/com/pentalog/pentastagiu/homework2/ConsoleApp/Basket.java

```diff
@@ -0,0 +1,21 @@
+package com.pentalog.pentastagiu.homework2.ConsoleApp;
+
+public class Basket {
+    private static float TotalPrice ;
```

Comment:
numele fieldurilor si metodelor dintr-o clasa trebuie sa respecte conventia camelCase (totalPrice, addProduct, ...) - observatie valabila pt toate clasele adaugate in pull request